tests/App/AppKernel.php

```diff
@@ -1,14 +1,25 @@
 <?php
 
+declare(strict_types=1);
+
 /*
  * This file is part of the Liip/FunctionalTestBundle
  *
  * (c) Lukas Kahwe Smith <smith@pooteeweet.org>
  *
  * This source file is subject to the MIT license that is bundled
  * with this source code in the file LICENSE.
+ */
+
+namespace Liip\FunctionalTestBundle\Tests\App;
+
+/*
+ * This file is part of the Liip/FunctionalTestBundle
  *
- * @see http://www.whitewashing.de/2012/02/25/symfony2_controller_testing.html
```

Comment:
This is getting lost, but it's probably outdated since it's from 2012...

Reply:
Yes, this can be removed. (I added this link 2 years ago as a reference but most of the code in the blog post is not necessary anymore)